packages/plugins/users-permissions/admin/src/pages/EmailTemplates/components/EmailTable.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import { Table, Thead, Tbody, Tr, Td, Th } from '@strapi/parts/Table';
+import { VisuallyHidden } from '@strapi/parts/VisuallyHidden';
+import { Text, TableLabel } from '@strapi/parts/Text';
+import { IconButton } from '@strapi/parts/IconButton';
+import EditIcon from '@strapi/icons/EditIcon';
+import Reload from '@strapi/icons/Reload';
+import CheckIcon from '@strapi/icons/CheckIcon';
+import { getTrad } from '../../../utils';
+
+const EmailTable = ({ canUpdate, onEditClick }) => {
```

Comment:
You need to use the data to build the table body, you can have more than 2 email templates.